[Term Entry] Java Calendar getTimeZone() #3098

HamishHill-WK opened the pull request:
### Description

- Added `getTimeZone` directory to `content/java/concepts/calendar/terms/getTimeZone`

- Added new term entry file to `content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md` for Java Calendar Class
Closes  #3098 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=3157) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=3157) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=3157) it.</sub>

SSwiniarski commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
```diff
@@ -0,0 +1,51 @@
+---
+Title: '.getTimeZone()'
+Description: 'Returns the current time-zone of this Calendar.'
```
```suggestion
Description: 'Returns the current time-zone of a given Calendar.'
```

SSwiniarski commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
```diff
@@ -0,0 +1,51 @@
+  - 'paths/computer-science'
+---
+
+
+The **getTimeZone()** method in Calendar class is used to return the current time-zone of this Calendar.
```
```suggestion
The **`.getTimeZone()`** method in Calendar class is used to return the current time-zone of a given Calendar.
```

SSwiniarski commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,51 @@
+```
+
+## Example
+
````
We want a introductory sentence to summarize what the example is doing.

SSwiniarski commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,51 @@
+```pseudo
+TimeZone time_zone = myCalendar.getTimeZone();
+```
+
````
```suggestion
We want a sentence after the syntax block explaining it. i.e. what `myCalendar` and `time_zone` are.
```

SSwiniarski reviewed (changes requested):
I did a first review and left a few comments. Please note that you'll need to change your PR so it isn't a draft, otherwise we cannot merge it.

SSwiniarski commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,52 @@
+```pseudo
+TimeZone time_zone = myCalendar.getTimeZone();
+```
+myCalendar is a Calendar class object. The Calendar is an abstract class that provides methods for converting between a specific instant in time and a set of calendar fields such as YEAR, MONTH, DAY_OF_MONTH, HOUR, and so on, and for manipulating the calendar fields, such as getting the date of the next week.
+time_zone is a TimeZone class Object. The Java TimeZone class is a class that represents time zones, and is helpful when doing calendar arithmetics across time zones. The java.util.TimeZone class is used in conjunction with the java.util.Calendar class.
````
```suggestion

- `myCalendar` is a `Calendar` class object for which the time zone is being retrieved.. 
- `time_zone` is a `TimeZone` class object returned from the `.getTimeZone()` method.
```

This is all we need for the syntax section.

SSwiniarski commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
```diff
@@ -0,0 +1,52 @@
+myCalendar is a Calendar class object. The Calendar is an abstract class that provides methods for converting between a specific instant in time and a set of calendar fields such as YEAR, MONTH, DAY_OF_MONTH, HOUR, and so on, and for manipulating the calendar fields, such as getting the date of the next week.
+time_zone is a TimeZone class Object. The Java TimeZone class is a class that represents time zones, and is helpful when doing calendar arithmetics across time zones. The java.util.TimeZone class is used in conjunction with the java.util.Calendar class.
+
+The following code creates a Calendar class object and then assigns the object's current time zone to a TimeZone object. The result is then printed to the screen. 
```
```suggestion
## Example

The following code creates a Calendar class object and then assigns the object's current time zone to a TimeZone object. The result is then printed to the screen. 
```
You deleted the example header.

SSwiniarski reviewed (changes requested):
@HamishHill-WK Two more comments.

SSwiniarski reviewed (approved):
@HamishHill-WK Looks good for a second review.

caupolicandiaz commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
```diff
@@ -0,0 +1,55 @@
+  - 'paths/computer-science'
+---
+
+The **`.getTimeZone()`** method in Calendar class is used to return the current time-zone of a given Calendar.
```
```suggestion
The **`.getTimeZone()`** method in the `Calendar` class is used to return the current time-zone of a given calendar.
```

caupolicandiaz commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,55 @@
+TimeZone time_zone = myCalendar.getTimeZone();
+```
+
+- `myCalendar` is a `Calendar` class object for which the time zone is being retrieved.. 
````
```suggestion
- `myCalendar` is a `Calendar` class object for which the time zone is being retrieved.
```

caupolicandiaz commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,55 @@
+## Syntax
+
+```pseudo
+TimeZone time_zone = myCalendar.getTimeZone();
````
```suggestion
time_zone = myCalendar.getTimeZone();
```

caupolicandiaz commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
```diff
@@ -0,0 +1,55 @@
+
+## Example
+
+The following code creates a Calendar class object and then assigns the object's current time zone to a TimeZone object. The result is then printed to the screen. 
```
```suggestion
The following code creates demonstrates the use of the `.getTimeZone()` method. 
```

caupolicandiaz commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,55 @@
+
+The following code creates a Calendar class object and then assigns the object's current time zone to a TimeZone object. The result is then printed to the screen. 
+
+```java
````
update the code to use 2 spaces for indentations. Tabs are not used in docs

caupolicandiaz commented on content/java/concepts/calendar/terms/getTimeZone/getTimeZone.md:
````diff
@@ -0,0 +1,55 @@
+
+```
+
+The above code will output:
````
```suggestion
The above code will return an output similar to:
```

caupolicandiaz reviewed (changes requested):
@HamishHill-WK I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

HamishHill-WK commented:
@caupolicandiaz thank you for your review, I've updated the code example to use spaces instead of tabs and implemented the other changes you requested

caupolicandiaz reviewed (approved):
@HamishHill-WK @SSwiniarski LGTM
